vsutil.py

```diff
@@ -166,3 +166,12 @@ def is_image(filename: str) -> bool:
     Returns true if a filename refers to an image.
     """
     return mimetypes.types_map.get(os.path.splitext(filename)[-1], "").startswith("image/")
+
+
+def clip_sum(*clips: List[vs.VideoNode]) -> vs.VideoNode:
+    """
+    Concatenates all input clips.
+    """
+    if len(clips) == 1:
+        return clips[0]
+    return sum(clips[1:], clips[0])
```

Comment:
`return core.std.Splice(clips)`

which is faster than `clip_a+clip_b+clip_c+...` as this evaluates to `Splice([Splice(Splice([a, b]), c], ...)`

Reply:
I completely forgot std.Splice takes varargs. In that case, we can just drop this PR altogether since it adds basically no value.

Reply:
And for completeness' sake, `core.std.Splice(*iterable)` is the iterable version of it.